docs/cloud.md

```diff
@@ -0,0 +1,69 @@
+# Deploying Antrea on a cloud provider
+
+We support running Antrea in pass-through mode in aks and eks.
+
+## Pre-requisites
+1. to run eks cluster, install and configure aws cli, see
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html, and
+   https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html
+2. to run aks cluster, install and configure azure cli,
+   https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+3. install terraform
+   see https://learn.hashicorp.com/terraform/getting-started/install.html
+
+## Create an aks cluster via terraform
+create or obtain a azure service principal. And set the following environment variables from the service principal. For instance in .bashrc
+```bash
+export TF_VAR_aks_client_id=YOUR_SERVICE_PRINCIPAL_ID
+export TF_VAR_aks_client_secret=YOUR_SERVICE_PRINCIPAL_ID
+export TF_VAR_aks_resource_group_name=YOUR_RESOURCE_GROUP_NAME
+```
+install Antrea cloud tools
+```bash
+make install-cloud-tool
+```
+create aks cluster
+```bash
+~/terraform/aks
+```
+interact with aks cluster
+```
+aks kubectl ... // issue kubectl commands to aks cluster
```

Comment:
I feel like I'm missing something here. Why do I no longer have to use the full `~/terraform/aks` path here? As indicated above, I think we should not install these scripts to `~/terraform/aks`.

Reply:
sure I can added the full path.